### DISCUSSION TITLE
App seems to lose all styling

### DISCUSSION THREAD

**Discussion by derekgould** [gluestack/gluestack-ui#2568] [General]
Trying to use Gluestackv2 in an expo app, with auth and expo router. At a certain point app seems to lose all styling.

I've pasted two screenshots; the first shows how the login page is supposed to look. After implementing auth with a logout, for some reason the page loses all styling as shown in the second screenshot.

Any ideas?
![Simulator Screenshot - iPhone 16 - 2024-11-05 at 14 47 08](https://github.com/user-attachments/assets/9436357f-68db-4278-904e-975a93beffb5)
![Simulator Screenshot - iPhone 16 - 2024-11-06 at 11 15 19](https://github.com/user-attachments/assets/4d6bf343-52e7-4679-a345-497ae14c8652)

**Comment by marcelino-borges** [comment]
I kind of had this issue as well.
And I also created a discussion here, but no answer so far 
lol
